### PR TITLE
Use golang 1.5.1

### PR DESCRIPTION
**Pull request by grobie**
@juliusv

### CODE REVIEW

**Comment by juliusv** [comment]
Thanks! When upgrading to Go 1.5.1, this also needs to be removed: https://github.com/prometheus/utils/blob/master/Makefile.COMMON#L47-L49

...because the Mac Go 1.5.1 download links don't contain the OS X version anymore.

**Comment by grobie** [comment]
Updated. People wanting to install go < 1.5 will have to set GOPKG now.

File: Makefile.COMMON [hunk 1]
```diff
@@ -44,11 +44,7 @@ SRC    ?= $(shell find . -type f -name "*.go" ! -path "./.build/*")
...
 GOARCH ?= $(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m)))
 
-ifeq ($(GOOS),darwin)
-	RELEASE_SUFFIX ?= -osx$(shell sw_vers -productVersion)
-endif
-
-GO_VERSION ?= 1.4.2
+GO_VERSION ?= 1.5.1
 GOURL      ?= https://golang.org/dl
 GOPKG      ?= go$(GO_VERSION).$(GOOS)-$(GOARCH)$(RELEASE_SUFFIX).tar.gz
```
Comment (juliusv) [inline Makefile.COMMON:49]: Also remove the `RELEASE_SUFFIX` here?

Comment (grobie) [inline Makefile.COMMON:49]: Doh. Well it gives an easy way to use 1.4 in projects ;). I won't have
access to a computer the next two days.

**Comment by juliusv** [comment]
Alrighty, I'll just hot-patch it myself :)

**Pull request merged**